concordium-sdk/src/main/java/com/concordium/sdk/responses/transactionstatus/BakerSetOpenStatus.java

```diff
@@ -0,0 +1,33 @@
+package com.concordium.sdk.responses.transactionstatus;
+
+import com.concordium.sdk.transactions.AccountAddress;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.ToString;
+
+/**
+ * The Baker updated its {@link OpenStatus}
+ */
+@ToString
+@Getter
+public class BakerSetOpenStatus extends AbstractBakerResult {
+
+    /**
+     * The updated {@link OpenStatus} for the baker.
+     */
+    private final OpenStatus openStatus;
+
+    @JsonCreator
+    BakerSetOpenStatus(@JsonProperty("bakerId") String bakerId,
```

Comment:
Are you sure the baker id is a string? It should be a Word64.

Reply:
Well.. It successfully parsed it :) 

I will change this to reflect the real type.